tests/kernel/fatal/message_capture/prj.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_LOG=y
+CONFIG_LOG_MINIMAL=y
```

Comment:
Is this still need by current method of testing exception?

Reply:
@enjiamai  Hi, enjia. CONFIG_LOG is necessary,  because in fatal.c, the message is dump with LOG_ERR. This marco "CONFIG_LOG=y" enable the log function.